src/lib/Event/Listener/LoginListener.php

```diff
@@ -100,15 +98,20 @@ public function onSecurityInteractiveLogin(InteractiveLoginEvent $event): void
         }
     }
 
+    private function getCustomerId(): string
+    {
+        return (string) $this->configResolver->getParameter(
+            'authentication.customer_id',
+            Parameters::NAMESPACE
+        );
+    }
+
     /**
      * Returns notification API end-point.
      */
     private function getNotificationEndpoint(): string
```

Comment:
```suggestion
    private function getNotificationEndpoint(string $endpoint, int $customerId): string
```

We get customerId earlier and it can be passed here. Maybe we should do the same with `.event_tracking.endpoint` to be consistent? Extract it to separate method and pass here as argument.

Reply:
I changed this function a bit to generate a URI